### PR TITLE
1288626: Does not report pool ids as serial numbers, ignore duplicates

### PR DESCRIPTION
**Pull request by cnsnyder**

### CODE REVIEW

File: src/subscription_manager/managercli.py [hunk 1]
```diff
@@ -54,7 +54,8 @@
         parse_baseurl_info, format_baseurl, is_valid_server_info, \
         MissingCaCertException, get_client_versions, get_server_versions, \
         restart_virt_who, get_terminal_width, print_error, \
-        ProductCertificateFilter, EntitlementCertificateFilter
+        ProductCertificateFilter, EntitlementCertificateFilter, \
+        unique_list_items
```
Comment (wottop) [inline src/subscription_manager/managercli.py]: Put up a few lines ith the other method names. More readable.

File: test/test_utils.py [hunk 2]
```diff
@@ -8,7 +8,8 @@
     parse_baseurl_info, format_baseurl, \
     get_version, get_client_versions, \
     get_server_versions, friendly_join, is_true_value, url_base_join,\
-    ProductCertificateFilter, EntitlementCertificateFilter
+    ProductCertificateFilter, EntitlementCertificateFilter, \
+    unique_list_items
```
Comment (wottop) [inline test/test_utils.py]: Place with other methods

**Comment by wottop** [comment]
Other than the minor comments above, ack.

**Comment by cnsnyder** [comment]
@wottop I have updated the PR according to your comments above.

**Comment by wottop** [comment]
ack

**Pull request merged**